extension/src/shared/view/fields/general/LayerDatasetStoryField.tsx

```diff
@@ -0,0 +1,69 @@
+import { Pagination, styled, paginationItemClasses } from "@mui/material";
+import { useAtom } from "jotai";
+import { FC, useCallback, useEffect, useState } from "react";
+import Markdown from "react-markdown";
+
+import { ParameterList } from "../../../../prototypes/ui-components";
+import { useCamera } from "../../../reearth/hooks";
+import { CameraPosition } from "../../../reearth/types";
+import { DatasetStoryField } from "../../../types/fieldComponents/general";
+import { WritableAtomForComponent } from "../../../view-layers/component";
+
+const Wrapper = styled("div")(({ theme }) => ({
+  padding: theme.spacing(1),
+  fontSize: theme.typography.body2.fontSize,
+  [`img`]: {
+    maxWidth: "100%",
+  },
+}));
```

Comment:
We already have same component in `LegendDescriptionSection`, `LayerLayerDescriptionField`, and this component, so I think we should make common component in `ui-components`.

Reply:
Updated! Thank you!